### PR TITLE
Feature/board functional positive

### PR DESCRIPTION
**Pull request by raulchoque-eng**

### CODE REVIEW

File: src/test/resources/feature/board/board_funtional_positive.feature [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+        }
+        """
+    When I send the request
+    Then I get a 200 status code as response
+    And  I verify the response schema with Board
+    Examples: board with labelNames/blue value updating
+    | label_Name_blue                   |
+    | b                                 |
+    | new labelNames/blue with big name |
+    | 1$b*                              |
```
Comment (MadayAlcala) [inline src/test/resources/feature/board/board_funtional_positive.feature]: Plase add a final line.

Comment (raulchoque-eng) [reply src/test/resources/feature/board/board_funtional_positive.feature]: Thanks, this mistake was fixed

**Review by MadayAlcala — APPROVED** [review]
Good job!

File: src/test/resources/feature/board/board_funtional_positive.feature [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+
+  @delete-board
+  Scenario Outline: Create a new Board with and without list
+    Given I create a POST request to /boards/ endpoint
+    And I set up the data:
+      """
+        {
+           "name":"[<name_board>]",
+           "defaultLists": <default_lists>
```
Comment (aburgosu) [inline src/test/resources/feature/board/board_funtional_positive.feature]: Delete white space after ':'

File: src/test/resources/feature/board/board_funtional_positive.feature [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+    | New board  | false         |
+
+  @delete-board
+  Scenario Outline: Create a new Board with description
+    Given I create a POST request to /boards/ endpoint
+    And I set up the data:
+      """
+        {
+           "name":"[<name_board>]",
+           "desc": "<description>"
```
Comment (aburgosu) [inline src/test/resources/feature/board/board_funtional_positive.feature]: Delete white space after ':'

File: src/test/resources/feature/board/board_funtional_positive.feature (hunk 2 above)
Comment (raulchoque-eng) [reply src/test/resources/feature/board/board_funtional_positive.feature]: thanks, this mistake was fixed

File: src/test/resources/feature/board/board_funtional_positive.feature (hunk 3 above)
Comment (raulchoque-eng) [reply src/test/resources/feature/board/board_funtional_positive.feature]: thanks, this mistake was fixed

**Review by aburgosu — APPROVED** [review]

**Pull request merged**